src/mavedb/lib/authorization.py

```diff
@@ -21,6 +21,15 @@ async def require_current_user(user: Optional[User] = Depends(get_current_user))
     return user_value
 
 
+async def require_current_user_with_email(user: User = Depends(require_current_user)) -> User:
+    if user.email is None:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
```

Comment:
wait, is this 400 here and 403 below?

Reply:
I originally made it 403 but then I started second guessing and thought 400 would make more sense. I'll make sure the response codes are consistent before I open it.